src/sentry/organizations/services/organization/impl.py

```diff
@@ -475,7 +475,7 @@ def get_or_create_team_member(
 
     def update_membership_flags(self, *, organization_member: RpcOrganizationMember) -> None:
         model = OrganizationMember.objects.get(id=organization_member.id)
-        model.flags = self._deserialize_member_flags(organization_member.flags)
+        model.flags = self._deserialize_member_flags(organization_member.flags)  # type: ignore[assignment]  # TODO: make BitField a mypy plugin
```

Comment:
the issue here is HC converts the bitfield to an integer over the wire and the assignment then confuses our workaround for bitfields

I plan to follow up by making bitfield an actual mypy plugin which understands the assignments and values properly here